installer/README.md

```diff
@@ -116,12 +116,13 @@ The following are the list configurable parameters of Volcano Chart and their de
 |`custom.admission_ns`|nodeSelector for Admission pods|`~`|
 |`custom.controller_ns`|nodeSelector for Controller pods|`~`|
 |`custom.scheduler_ns`|nodeSelector for Scheduler pods|`~`|
-|`custom.admission_podLabels`|Pod labels for Admission pods|`~`|
-|`custom.controller_podLabels`|Pod labels for Controller pods|`~`|
-|`custom.scheduler_podLabels`|Pod labels for Scheduler pods|`~`|
-|`custom.admission_labels`|Labels for Admission deployment and job|`~`|
-|`custom.controller_labels`|Labels for Controller deployment|`~`|
-|`custom.scheduler_labels`|Labels for Scheduler deployment|`~`|
+|`custom.admission_podLabels`|Pod labels for Admission pods|`{}`|
+|`custom.controller_podLabels`|Pod labels for Controller pods|`{}`|
+|`custom.scheduler_podLabels`|Pod labels for Scheduler pods|`{}`|
+|`custom.admission_labels`|Labels for Admission deployment and job|`{}`|
```

Comment:
Welcome! Why change default value to `~`?

Reply:
Hi, @Monokaix !

Sorry for this, for easirer merge I decided to set this to `{}` by default. However on a second thought it is also pretty easy to support `~` in merge, so I updated the PR to remove this change. Please let me know if anything else is needed.